.devcontainer/docker-compose-local.yaml

```diff
@@ -97,6 +97,8 @@ services:
       - RAY_WORKER_GPUS_FRACTION=0
     volumes:
       - ../:/mzai
+    extra_hosts:
```

Comment:
I think we should add an extra note of caution here. (Since this modifies localhost networking)

Reply:
+100